extensions/cornerstone/src/services/ViewportService/Viewport.ts

```diff
@@ -241,11 +249,12 @@ class ViewportInfo {
   }
 
   private mapDisplaySetOptions(
-    publicDisplaySetOptions: Array<PublicDisplaySetOptions>
+    displaySetSelectors: Array<DisplaySetSelector> = [{}]
```

Comment:
Line 150 is calling `mapDisplaySetOptions` with an `Array<PublicDisplaySetOptions>`, yet the method itself accepts an `Array<DisplaySetSelector>`. What am I missing?

Reply:
The call hierarchy changed sometimes to be display set selectors and sometimes options, so updated the code to handle both.